index.js

```diff
@@ -0,0 +1,50 @@
+import { exec } from 'child_process';
+import readline from 'readline';
+
+import { help, create, show, rename, del } from './command';
+
+exec('mkdir output', (err, stdout, stderr) => { stderr });
+
+const rl = readline.createInterface(process.stdin, process.stdout);
```

Comment:
The variable `rl` should follow a better naming convention, say `readLine` maybe 🤷‍♂️